test/e2e/config/ironic.yaml

```diff
@@ -27,6 +27,7 @@ variables:
   BOOT_MAC_ADDRESS: "00:60:2f:31:81:01"
   IMAGE_URL: "http://192.168.222.1/cirros-0.6.2-x86_64-disk.img"
   IMAGE_CHECKSUM: "c8fc807773e5354afe61636071771906"
+  CERT_MANAGER_VERSION: "v1.13.0"
```

Comment:
This needs to be added in `fixture.yaml` also!
(We will have automated tests to check the fixture tests when #1437 is merged.)

Reply:
Done.